### PR TITLE
Adds Overmap Special: Nursing Home

### PR DESCRIPTION
**Pull request by pshepjr**
#### Summary
Content "Adds Overmap Special, Nursing Home"

#### Purpose of change

This PR adds a new overmap special, the Nursing Home. [ This is a recreation of #58805 ]

#### Describe the solution

This adds a new generic nursing home location with two brand new enemies, the Zeriatric and Feral Granny. Additionally, there is a 5% chance for a variant of this location to spawn, the Shady Hills Nursing Home. This rare variant has a secret underground lab along with a third new enemy, Big Pappy.

The nursing home generally spawns close to (or even right next to) a city. The generic version has an occurrence of 0 to 3.

I've expanded more in detail below.

#### Describe alternatives you've considered

None

#### Testing

Many, many game restarts checking to see if it spawned, and force spawning in the console. Also see previously mentioned PR #58805 with a lot of feedback that was incorporated here.

#### Additional context

**MONSTERS ADDED**

Zeriatric:
> "A hunched, decrepit, geriatric zombie. Its dentures seem to be falling out."
> 
> Zeriatrics are weak, frail, slow, and deaf. They also can't bite you. Their vision range is extremely poor, even in the daylight. All in all, they aren't much of a threat. That being said - they grab and scratch more often than the standard zombie.
> 
> Loot consists of basic clothing, with a chance of glasses and/or a cane.

Feral Granny:

> "A sweet old lady in a wheelchair. As you move closer, she offers you some cookies. Do you want a cookie?"
> 
> Feral Grannies are exactly as the name would imply. Weaker and frailer than their feral human counterparts, they chase down the survivor in their wheelchair while trying to grab onto them and wail loudly. Because of their higher speed, they can be difficult to run away from. They also tend to constantly spout nonsense (Parrot skill) such as "Do you want  a cookie, honey?" and "You're looking feverish. Let grandma make you some soup". Their unpulped corpse will become a crawling zombie.
> 
> Loot consists of basic clothing, their wheelchair, and a chance of glasses, dentures, or a cane. Sometimes drops a cookie.

Big Pappy:

> "A gigantic, monstrous old man with a crazed, anguished face. He carries a large cane, and seems intent to discipline you with it."
> 
> Encountered only in the Shady Hills variant, Big Pappy is intended to be somewhat of a boss encounter for the survivor.  Stat wise, Big Pappy is the feral human version of a zombie wrestler, but with some additional adjustments, including the Impale ability. To quote him, "I'm 'bout to walk 15 miles both ways across your ass." He is flanked by two grannies, who become quite the nuisance by grabbing onto the player and making it difficult for them to run. Failing to run from Big Pappy can prove to be a fatal mistake, as he enjoys tossing  the survivor around like a ragdoll.
> 
> Loot consists of a bathrobe, footrags, and a cane. His unpulped corpse becomes a zombie wrestler.
> 

Other enemies found here: Regular zombies, fat zombies, crawlers, Boomers (because obviously..), medical zombies, scientist zombies [in the lab], technician zombies [in the lab].

**ITEMS/VEHICLES ADDED**

Small Hospital Bed: It's the hospital bed, but only one tile.

Folded Folding Wheelchair: Lootable item that deploys the existing Folding Wheelchair vehicle. This is part of the Feral Granny loot table.

Newspaper article [months-old]: POLICE TO INVESTIGATE SHADY HILLS.  There will be a "thorough investigation" into the Shady Hills nursing home, owned by Shady Hills Corporation, said police Monday. The announcement came following several lawsuits launching against the home by family members of residents for alleged elder abuse and neglect.

**LOOT**

In the generic nursing home, the most valuable loot to the survivor would likely be the low level medical supplies such as bandages, antiseptic, etc. Occasionally, some manuals can be found in the bookshelves, but its rare. Food and kitchen supplies can be found in the back. The employee break room can have a wide variety of items in the lockers, but nothing too crazy.

In the Shady Hills variant, the underground lab has typical chemistry/science supplies. Getting past Big Pappy allows one to access the locked storage, which is guaranteed to contain a sizeable amount of medical ampoules and rivtech injectors. You can also potentially find mutagens and some other random science/chemistry items.

**STORY**

The generic nursing home is just a standard nursing home. But for the Shady Hills variant, the survivor can come across a loose story through environmental observation and using the computers scattered across the home. 

Shady Hills nursing home - at first glance to any survivor, it's an average nursing home location. There's bits of trash scattered across the floor, but surely that's just a result of the chaos of the cataclysm, right? Using various consoles found across the home, the survivor uncovers emails between multiple employees of differing ranks/roles in the home. Most of it seems standard enough - discussion about budget, performance evaluations, gossip - but a few of the emails are admittedly bizarre, hinting at the nursing home experiencing legal and financial trouble, and perhaps being involved in something far more dubious. Unfortunately, it seems most of the database isn't functional, leaving the survivor unable to come across much of anything concrete.

If the survivor is perceptive enough, they may stumble across a hidden closet with a non-functioning elevator. Luckily, there's a ladder. Sickeningly - the ladder leads to a lab where, apparently, aa group of 'doctors' were performing questionable 'experiments' using the residents as guinea pigs.

If the survivor uses the console of the head researcher, they will discover that Shady Hills had been using the residents to research anti-aging technology. But it seems one particular resident did not react as they expected... He not only needed to be locked down in the lab, but sedated using an extremely high level of elephant tranquilizer. What happens if that old man wakes up...?

It seems the events of the cataclysm ended up taking priority over whatever kind of experiments were happening here...

The front of the nursing home:
Front Side

The back of the nursing home:
backside

From the side:
sideview

The roof:
roof

Courtyard:
courtyard

View on overmap (it's rotated 180 degrees in this example):
Overmap

Shady Hills variant:
shady hills - front

Shady hills showing hidden lab entrance:
shady hills - back

Secret lab elevator shaft:
shady hills - secret passage

Part 1 of the lab:
shady hills - lab

Part 2 of the lab, featuring Big Pappy:
shady hills - lab2

Example from the receptionist's PC:
Console1

Example from the SVP Operations' PC:
console2

Shouldn't this kind of corporation have better security? You'd think so...
console3

### CODE REVIEW

File: data/json/items/tool/deployable.json [hunk 1]
```diff
@@ -367,5 +367,23 @@
     "symbol": "7",
     "color": "yellow_red",
     "use_action": { "type": "deploy_furn", "furn_type": "f_manual_tire_changer" }
+  },
+  {
+    "id": "folded_wheelchair",
+    "type": "TOOL",
+    "name": { "str": "folded Foldable wheelchair" },
```
Comment (Night-Pryanik) [inline data/json/items/tool/deployable.json:374]:
```suggestion
    "name": { "str": "folded wheelchair" },
```

File: data/json/items/tool/deployable.json [hunk 2]
```diff
@@ -367,5 +367,23 @@
...
+    "description": "A folded Foldable wheelchair.",
```
Comment (Night-Pryanik) [inline data/json/items/tool/deployable.json:375]:
```suggestion
    "description": "A folded wheelchair.",
```

File: data/json/items/tool/deployable.json [hunk 3]
```diff
@@ -367,5 +367,23 @@
...
+    "weight": "10586 g",
+    "volume": "13500 ml",
+    "longest_side": "0 cm",
```
Comment (Night-Pryanik) [inline data/json/items/tool/deployable.json:378]: I guess this should be set to something?

File: data/json/monsters/feral_humans.json [hunk 4]
```diff
@@ -1019,5 +1019,119 @@
       "PATH_AVOID_DANGER_1",
       "DROPS_AMMO"
     ]
+  },
+  {
+    "id": "mon_feral_big_pappy",
+    "type": "MONSTER",
+    "name": { "str": "Big Pappy", "str_pl": "Big Pappys" },
```
Comment (Night-Pryanik) [inline data/json/monsters/feral_humans.json:1026]:
```suggestion
    "name": { "str": "Big Pappy" },
```
`s` is added automatically.

File: data/json/overmap/overmap_special/specials.json [hunk 5]
```diff
@@ -7697,5 +7697,99 @@
...
+      { "point": [ 0, 0, 1 ], "overmap": "nursing_home_1_roof_north" },
+      { "point": [ 1, 0, 1 ], "overmap": "nursing_home_2_roof_north" },
+      { "point": [ 2, 0, 1 ], "overmap": "nursing_home_3_roof_north" },
+      { "point": [ 0, 1, 1 ], "overmap": "nursing_home_4_roof_north" },
+      { "point": [ 1, 1, 1 ], "overmap": "nursing_home_5_roof_north" },
+      { "point": [ 2, 1, 1 ], "overmap": "nursing_home_6_roof_north" }
+    ],
+    "connections": [ { "point": [ 3, 3, 0 ], "terrain": "road", "connection": "local_road", "from": [ 3, 2, 0 ] } ],
+    "city_distance": [ 0, 5 ],
+    "occurrences": [ 0, 3 ],
```
Comment (Night-Pryanik) [inline data/json/overmap/overmap_special/specials.json:7792]: IMO up to three nursing homes per overmap seems excessive. I'd limit it to max 1 per overmap.

File: data/json/npcs/TALK_NURSING_HOME.json [hunk 6]
```diff
@@ -0,0 +1,524 @@
...
+      { "text": "View Patient Medical History", "topic": "COMP_NURSING_HOME_SCI_ERROR3" },
+      { "text": "View Dosages", "topic": "COMP_NURSING_HOME_SCI_ERROR3" },
+      { "text": "View Mutagenic Serums", "topic": "COMP_NURSING_HOME_SCI_ERROR3" },
+      { "text": "RETURN TO PROJECTS", "topic": "COMP_NURSING_HOME_SCI_PROJECTS" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "COMP_NURSING_HOME_SCI_PROJECTB",
+    "dynamic_line": "PROJECT CODE NAME: 'Big Pappy'\n PROJECT RESIDENT: Nina Palmero\n PROJECT STATUS: Active\n PROJECT START DATE: xx-xx-xxxx\n\n Entry Log 1: Started subject on assigned dosage.  WIll monitor over 7 days.\n Entry Log 2: Subject added to growing list of recent residents that are rejecting the dosage due to an unknown cause.  Attempting supplement of benzoylmethylecgonine to study possible results.  Will monitor over 7 days.  Mainting subject in general population.\n Entry Log 3: Subject appears to have become obsessed with baked goods, unknown reason.",
```
Comment (Night-Pryanik) [inline data/json/npcs/TALK_NURSING_HOME.json:491]:
```suggestion
    "dynamic_line": "PROJECT CODE NAME: 'Big Pappy'\n PROJECT RESIDENT: Nina Palmero\n PROJECT STATUS: Active\n PROJECT START DATE: xx-xx-xxxx\n\n Entry Log 1: Started subject on assigned dosage.  Will monitor over 7 days.\n Entry Log 2: Subject added to growing list of recent residents that are rejecting the dosage due to an unknown cause.  Attempting supplement of benzoylmethylecgonine to study possible results.  Will monitor over 7 days.  Maintaining subject in general population.\n Entry Log 3: Subject appears to have become obsessed with baked goods, unknown reason.",
```

File: data/json/items/tool/deployable.json (hunk 3 above)
Comment (pshepjr) [reply data/json/items/tool/deployable.json]: Interesting - I thought I had done this, I even specifically remember looking up the average size of a wheelchair. Guess I messed it up somehow, thanks for the catch. I set it to 92cm (average height is 36 inches which converts to 91.44 cm).

File: data/json/monsters/feral_humans.json (hunk 4 above)
Comment (pshepjr) [reply data/json/monsters/feral_humans.json]: Yep - it failed it again haha

Comment (anothersimulacrum) [reply data/json/monsters/feral_humans.json]: Possibly because it ends in a `y`?

Comment (pshepjr) [reply data/json/monsters/feral_humans.json]: Let me know if you want me to re-add it.

Comment (pshepjr) [reply data/json/monsters/feral_humans.json:1026]: Done!

**Comment by Night-Pryanik** [comment]
Build is failing because of reasons not related to your PR.

**Comment by pshepjr** [comment]
Thank you for the clarification.

**Comment by pshepjr** [comment]
Hi there, is this still under review or should I mention it in discord as ready to merge? Not trying to rush just making sure I am on the right page :) Thanks!

**Pull request merged**